lib/less/contexts.js

```diff
@@ -79,6 +79,13 @@ contexts.Eval.prototype.isPathRelative = function (path) {
     return !/^(?:[a-z-]+:|\/)/i.test(path);
```

Comment:
I didn't dig into that regex earlier since the method name had me believe it was looking for things like `../../folder`, `~/folder`, or `/folder`. I didn't realize that it was doing this by checking that it was not something else.

So, I do see how it would fail for `data:image`, however `'data:image....` is getting passed in. Note the leading quotation mark. Would it make more sense to strip out that quotation mark in this method or require the caller to strip it out? Or is this an issue with the `Quoted` prototype?